backend/cmd/migrate/migrations/000005_create_topology_cache_table.down.sql

```diff
@@ -1 +1 @@
-DROP TABLE IF EXISTS topology_cache_data;
+DROP TABLE IF EXISTS topology_cache;
```

Comment:
We renamed this table a while back, but this reference was not updated. Bundling it with this PR.